apache-rat-core/src/test/java/org/apache/rat/IOptionsProvider.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ */
+package org.apache.rat;
+
+/**
+ * A list of methods that an OptionsProvider in a test case must support.
+ * Use of this interface can ensures consistent testing across the UIs.  Each method
+ * tests an Option from OptionCollection that must be implemented in the UI.
+ */
+public interface IOptionsProvider {
+    void addLicenseTest();
```

Comment:
I guess all of these methods need good Javadoc comments in case new options are added or older ones removed .....

Reply:
Could we make this package scoped as it is in tests only? At first I thought that this class is in the src/main-directory tree.

Reply:
Not possible as it is used in other modules (Ant/Maven plugin)